data/talkactions/scripts/attributes.lua

```diff
@@ -0,0 +1,47 @@
+function onSay(player, words, param)
+	if not player:getGroup():getAccess() then
+		return true
+	end
+
+	local position = player:getPosition()
+	position:getNextPosition(player:getDirection())
+
+	local tile = Tile(position)
+	if not tile then
+		player:sendTextMessage(MESSAGE_STATUS_CONSOLE_BLUE, "There is no tile in front of you.")
+		return false
+	end
+
+	local thing = tile:getTopVisibleThing(player)
+	if not thing then
+		player:sendTextMessage(MESSAGE_STATUS_CONSOLE_BLUE, "There is an empty tile in front of you.")
+		return false
+	end
+
+	local separatorPos = param:find(',')
+	if separatorPos then
+		attribute = string.trim(param:sub(0, separatorPos - 1))
+		value = string.trim(param:sub(separatorPos + 1))
+	else
+		player:sendTextMessage(MESSAGE_STATUS_CONSOLE_BLUE, string.format("Usage: %s attribute, value.", words))
+		return false
+	end
+
+	if thing:isItem() then
```

Comment:
would prefer for this to be
if not thing:isItem() then
return false
end

its more readable this way

Reply:
There will be new types added as soon as this is merged... 🙄

Reply:
I've updated the separatorPos check anyway